specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2019-04-01-preview/costmanagement.json

```diff
@@ -0,0 +1,1823 @@
+          "type": "string",
+          "readOnly": true
+        },
+        "display": {
+          "description": "The object that represents the operation.",
+          "properties": {
+            "provider": {
+              "description": "Service provider: Microsoft.CostManagement.",
+              "type": "string",
+              "readOnly": true
+            },
+            "resource": {
+              "description": "Resource on which the operation is performed: Dimensions, Query.",
+              "type": "string",
+              "readOnly": true
+            },
+            "operation": {
+              "description": "Operation type: Read, write, delete, etc.",
+              "type": "string",
+              "readOnly": true
+            }
+          }
+        }
+      }
+    },
+    "Scope": {
+      "type": "object",
+      "description": "The Scope model definition",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Scope id"
+        },
+        "name": {
+          "type": "string",
+          "description": "Scope name"
+        },
+        "type": {
+          "type": "string",
+          "description": "Scope type"
+        },
+        "childScope": {
+          "$ref": "#/definitions/Scope"
+        }
+      }
+    },
+    "ViewListResult": {
+      "description": "Result of listing views. It contains a list of available views.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The list of views.",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/View"
+          }
+        },
+        "nextLink": {
+          "description": "The link (url) to the next page of results.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "View": {
+      "description": "States and configurations of Cost Analysis.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ViewProperties",
+          "title": "View properties"
+        }
+      }
+    },
+    "ViewProperties": {
+      "type": "object",
+      "description": "The properties of the view.",
+      "properties": {
+        "version": {
+          "description": "View API version used to create the view.",
+          "type": "string"
+        },
+        "queryVersion": {
+          "description": "Query API version to use.",
+          "type": "string"
+        },
+        "displayName": {
+          "description": "User input name of the view. Required.",
+          "type": "string"
+        },
+        "scope": {
+          "description": "Cost Management scope to save the view on.",
+          "type": "string"
+        },
+        "createdOn": {
```

Comment:
this is most likely a read-only property. please update.

Reply:
Updated